migrations/2024-07-14-074318_enable-stopwords-in-textsearchable_index_col/up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE OR REPLACE aggregate tsvector_agg (tsvector) (
+  STYPE = pg_catalog.tsvector,
+  SFUNC = pg_catalog.tsvector_concat,
+  INITCOND = ''
+);
```

Comment:
what does this do? could probably use a comment :)

Reply:
This is an aggregation function that combines multiple rows of `tsvector` data into a single `tsvector` using the `tsvector` `concat` operator (`||`).

e.g.
```PgSql
WITH expected AS (
  SELECT
    'macro:1'::tsvector || 'any:1'::tsvector AS concat
),
data as (
  SELECT *
  FROM (
    VALUES
      ('macro:1' :: tsvector),
      ('any:1' :: tsvector)
  ) k(tv)
)
SELECT
  ( SELECT concat FROM expected ),
  ( SELECT tsvector_agg(tv) FROM data ) AS agg,
  ( SELECT concat FROM expected ) = (
    SELECT tsvector_agg(tv) FROM data
  ) AS is_eq;
```
```
      concat       |        agg        | is_eq
-------------------+-------------------+-------
 'any':2 'macro':1 | 'any':2 'macro':1 | t
(1 row)
```